R/MsBackend.R

```diff
@@ -294,55 +440,139 @@ setMethod("backendInitialize", signature = "MsBackend",
               object
           })
 
-## Data accessors
+#' @rdname MsBackend
+setMethod("backendMerge", "list", function(object, ...) {
```

Comment:
Combining more than two backends is key later in the `setBackend,Spectra` method, where I want to do the change of backends in parallel (by file). Reason: memory demand. We don't want to load all data into memory first and then save it to e.g. hdf5 files - better to do that on a per-file (or however you want to split the data) manner.

The name of the function is a different story. I used `backendMerge` somehow like we had the `backend*` methods in our previous implementation - but that's no longer needed. Regarding `merge`, I think we said we would like to use `c` instead of `merge`. So, I will change the name, but will not change the possibility to combine an arbitrary number of backends.